src/main/java/org/mdbenefits/app/submission/actions/HandleApplicationSigned.java

```diff
@@ -1,16 +1,55 @@
 package org.mdbenefits.app.submission.actions;
 
+import com.mailgun.model.message.MessageResponse;
 import formflow.library.config.submission.Action;
 import formflow.library.data.Submission;
+import formflow.library.data.SubmissionRepositoryService;
+import formflow.library.email.MailgunEmailClient;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.MessageSource;
 import org.springframework.stereotype.Component;
 
-
 @Slf4j
 @Component
 public class HandleApplicationSigned implements Action {
 
+    private final MessageSource messageSource;
+
+    private final MailgunEmailClient mailgunEmailClient;
+
+    private final SubmissionRepositoryService submissionRepositoryService;
+
+    public HandleApplicationSigned(MessageSource messageSource, MailgunEmailClient mailgunEmailClient,
+            SubmissionRepositoryService submissionRepositoryService) {
+        this.messageSource = messageSource;
+        this.mailgunEmailClient = mailgunEmailClient;
+        this.submissionRepositoryService = submissionRepositoryService;
+    }
+
     @Override
     public void run(Submission submission) {
+        sendEmailToApplicant(submission);
+    }
+
+    private void sendEmailToApplicant(Submission submission) {
+        String recipientEmail = (String) submission.getInputData().get("emailAddress");
+        if (recipientEmail == null || recipientEmail.isBlank()) {
```

Comment:
@vrajmohan - How would you feel about adding a check here to see what environment we are in. 

If, in staging or dev, maybe we can skip email sends, or override to send to a test mailing list instead?

We are testing a lot and I'm concerned that we might accidentally send emails out every once in a while.

Reply:
Good point, see comment below.